css-canvas-medical/styles.css

```diff
@@ -0,0 +1,225 @@
+h1,
+h2 {
+  font-family: 'Tiempos Headline', serif;
+  font-weight: 300;
+  margin: 0;
+}
+
+h3,
+p {
+  font-family: 'avenir', sans-serif;
+}
+
+h1 {
+  font-size: 64px;
+}
+
+h2 {
+  font-size: 46px;
+}
+
+h3 {
+  font-size: 28px;
+  line-height: 2;
+}
+
+p {
+  font-size: 24px;
+  line-height: 2;
+}
+
+.logo {
+  max-width: 220px;
+}
+
+.white {
+  color: white;
+}
+
+.lightgray {
+  color: #707070;
+}
+
+.darkgray {
+  color: #525351;
+}
+
+.row {
+  display: flex;
+}
+
+.rowpadding {
+  flex-basis: 100%;
+  padding-left: 7%;
+  padding-right: 7%;
+}
```

Comment:
This looks like something that should be a `column-full` class, and you should move the `padding` to a separate class you can apply along with this class.